contentpacks/khanacademy.py

```diff
@@ -881,8 +882,6 @@ def apply_dubbed_video_map(content_data: list, dubmap: dict, subtitles: list, la
             if dubmap.get(item.get("youtube_id", "")): # has a dubbed video
                 item.update(dubmap.get(item["youtube_id"]))
                 dubbed_count += 1
-            elif item["kind"] == NodeType.video and item["youtube_id"] not in subtitles: # no subtitles
```

Comment:
@rtibbles what are the implications of not filtering out video nodes if they're not dubbed and have no subtitles? This was the main reason there were no video nodes for `pt-BR`.